console/console.go

```diff
@@ -95,7 +98,25 @@ func GetConsolePluginCR(pluginName string, displayName string, consolePort int,
 //+kubebuilder:rbac:groups="",resources=services,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=console.openshift.io,resources=consoleplugins,verbs=*
 
-func InitConsole(client client.Client, odfPort int) error {
+func InitConsole(client client.Client, odfPort int ) error {
+	// Determine ocp version
+	clusterVersionList := configv1.ClusterVersionList{}
+	if err := client.List(context.TODO(), &clusterVersionList); err != nil {
+		return err
+	}
+	clusterVersion := ""
+	for _, version := range clusterVersionList.Items {
+		for _, history := range version.Status.History {
+			if history.State == configv1.CompletedUpdate {
+				clusterVersion = history.Version;
+			}
+		}
+	}
+	basePath := "release-4.9"
```

Comment:
add a newline above and remove newline from below.
Line 115 and 117 are logically same block.